backend/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -604,12 +604,20 @@ def _prep_zypp_repo_url(self, url):
         if self.proxy_pass:
             query_params['proxypass'] = self.proxy_pass
         if self.sslcacert:
-            query_params['ssl_capath'] = self.sslcacert
+            # Since Zypper only accepts CAPATH, we need to split the certificates bundle
+            # and run "c_rehash" on our custom CAPATH
+            _ssl_capath = os.path.dirname(self.sslcacert)
+            msg = "Preparing custom SSL CAPATH at {}".format(_ssl_capath)
```

Comment:
Hmm, you use the same directory where the sslcacert file is located.
What are typical directories ? 

Is see problems with cleaning them up later when it is just /tmp/ or any other normally used dir.
Or does the outside code copy it already to a special directory?

Reply:
The certificates files are written by `reposync` on `/var/cache/rhn/reposync/.ssl-certs/ORGNUMBER/` when you execute `spacewalk-repo-sync` targeting a channel that is using SSL certificates. The lifetime of those certificate files is only the current `reposync` run.

There is this cleanup mechanism which should wipe everything on there: https://github.com/uyuni-project/uyuni/blob/master//backend/satellite_tools/reposync.py#L352 

This cleanup mechanism is called everytime we call `reposync`: https://github.com/uyuni-project/uyuni/blob/master/backend/satellite_tools/spacewalk-repo-sync#L243

Reply:
perfect